charts/super-agent/Chart.yaml

```diff
@@ -3,12 +3,12 @@ name: super-agent
 description: Bootstraps New Relic' Super Agent
 
 type: application
-version: 0.0.1
+version: 0.0.1-beta
```

Comment:
I am not sure about the effects of merging this.. would a new release would be pushed again with 0.0.1-beta?

Reply:
No. The chart releaser will check [index.yaml looking for version 0.0.1-beta](https://github.com/newrelic/helm-charts/blob/gh-pages/index.yaml#L22528-L22558), and since it is available no release will be triggered.